content/content-strategy/set-up-governance/topic-takeaway.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: text/topictakeaway
+heading: Content strategy output
+firstcolumnheader: What you'll get
+secondcolumnheader: What to do
+firstcolumncontent: topic-takeaway-listitems.md
+previousheader: "Do before: Set goals and measure success"
+previoustext: Develop your strategy phase
+previouslink: /content-strategy/set-goals-measure-success/
+nextheader: "Do next: Remove content"
+nexttext: Clean up phase
+nextlink: /content-strategy/remove-content/
```

Comment:
Some of there are difficult to read maybe consider using `_` to split words

Reply:
I would split these by previous and next. First column and second column can be confusing for users entering in content.